test/src/unit-cppapi-filter.cc

```diff
@@ -139,10 +139,14 @@ TEST_CASE("C++ API: Filter lists on array", "[cppapi], [filter]") {
   FilterList a1_filters(ctx);
   a1_filters.set_max_chunk_size(10000);
   a1_filters.add_filter({ctx, TILEDB_FILTER_BYTESHUFFLE})
-      .add_filter({ctx, TILEDB_FILTER_BZIP2});
+      .add_filter({ctx, TILEDB_FILTER_BZIP2})
+      .add_filter({ctx, TILEDB_FILTER_CHECKSUM_MD5})
+      .add_filter({ctx, TILEDB_FILTER_CHECKSUM_SHA256});
```

Comment:
Is there a way to enforce an error upon reading? Perhaps manually corrupt some bit in the file or sth like that? If we do se, I would test MD5 and SHA256 separately, not in the same filter pipeline.